pstatus-report-sink-ktor/src/main/kotlin/gov/cdc/ocio/processingstatusapi/ReportManager.kt

```diff
@@ -239,13 +248,12 @@ class ReportManager: KoinComponent {
             this.dataStreamRoute = dataStreamRoute
             this.dexIngestDateTime = dexIngestDateTime
             this.jurisdiction = jurisdiction
-            this.senderId = senderId
             this.messageMetadata = messageMetadata
             this.stageInfo= stageInfo
             this.tags = tags
             this.data = data
-            this.jurisdiction = jurisdiction
```

Comment:
The jurisdiction should not be removed.

Reply:
Same. Removed the duplicated one. Somehow both were added twice.